### PR TITLE
Allow for multiple stars at the start of a C style comment

### PR DESCRIPTION
**Pull request by bantonsson**

### CODE REVIEW

**Comment by bantonsson** [comment]
Some people like comments like this you know :wink:
```
/**
 * Fancy header
 */
```

File: src/main/scala/de/heikoseeberger/sbtheader/SbtHeader.scala [hunk 1]
```diff
@@ -29,7 +29,7 @@ object SbtHeader extends AutoPlugin {
   object autoImport {
 
     object HeaderPattern {
-      val cStyleBlockComment = """(?s)(/\*(?!\*).*?\*/(?:\n|\r|\r\n)+)(.*)""".r
```
Comment (hseeberger) [inline src/main/scala/de/heikoseeberger/sbtheader/SbtHeader.scala]: I believe this defeats the purpose of the negative lookahead of a second "*" ;-)
So, I think your PR will lead to deleting JavaDoc/ScalaDoc comments and replacing these with the header.

Comment (bantonsson) [inline src/main/scala/de/heikoseeberger/sbtheader/SbtHeader.scala]: Or you can just trust that I ran the tests?

Comment (hseeberger) [inline src/main/scala/de/heikoseeberger/sbtheader/SbtHeader.scala]: So you agree that if your file doesn't yet contain a copyright header but starts with some ScalaDoc, then your changes will delete the ScalaDoc?

Comment (bantonsson) [inline src/main/scala/de/heikoseeberger/sbtheader/SbtHeader.scala]: Ok, then I'll add a new pattern that works they way I want, so you can pick and choose.

Comment (bantonsson) [inline src/main/scala/de/heikoseeberger/sbtheader/SbtHeader.scala]: I mean your definition of a copyright header is just the first comment in the file anyway.

Comment (hseeberger) [inline src/main/scala/de/heikoseeberger/sbtheader/SbtHeader.scala]: It's not a perfect definition, but a pragmatic approach that works well for ConductR and my pet projects. You can always use your own pattern, see the README for details. I won't include a pattern here which deletes JavaDoc or ScalaDoc.

Comment (bantonsson) [inline src/main/scala/de/heikoseeberger/sbtheader/SbtHeader.scala]: Ok, thanks.

**Pull request closed without merging**